test/Polly.Specs/Helpers/ObjectExtensions.cs

```diff
@@ -1,11 +0,0 @@
-﻿namespace Polly.Specs.Helpers;
-
-public static class ObjectExtensions
```

Comment:
I think this change as-is is undoing the intent of the helper method, which is to avoid the repetitive code to create dictionaries.

Instead maybe add a _different_ extension method that achieves the same effect.

Something like this:

```csharp
public static class DictionaryHelpers
{
    public static Dictionary<TKey, TValue> Create<TKey, TValue>(TKey key, TValue value)
        => new Dictionary<TKey, TValue>(1) { [key] = value };
}
```